pyproject.toml

```diff
@@ -26,29 +26,29 @@ dependencies = [
     'botocore>=1.31,<2',
     'compress-pickle>=1.2.0',
     'humanfriendly>=8.2',
-    "numpy>=1.20.0;python_version<'3.10'",
+    "numpy>=1.21.0;python_version<'3.10'",
     "numpy>=1.23.3,<2;python_version>='3.10' and python_version<'3.12'",
     "numpy>=1.26.0,<2;python_version>='3.12'",
-    "pandas>=1.1.3;python_version<'3.10'",
-    "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
+    "pandas>=1.4.0;python_version<'3.11'",
     "pandas>=1.5.0;python_version>='3.11' and python_version<'3.12'",
     "pandas>=2.1.1;python_version>='3.12'",
     'psutil>=5.7',
-    "scikit-learn>=0.24;python_version<'3.10'",
-    "scikit-learn>=1.1.3;python_version>='3.10' and python_version<'3.12'",
+    "scikit-learn>=1.0.2;python_version<'3.10'",
+    "scikit-learn>=1.1.0;python_version>='3.10' and python_version<'3.11'",
+    "scikit-learn>=1.1.3;python_version>='3.11' and python_version<'3.12'",
     "scikit-learn>=1.3.1;python_version>='3.12'",
-    "scipy>=1.5.4;python_version<'3.10'",
+    "scipy>=1.7.3;python_version<'3.10'",
     "scipy>=1.9.2;python_version>='3.10' and python_version<'3.12'",
     "scipy>=1.12.0;python_version>='3.12'",
     'tabulate>=0.8.3,<0.9',
-    "torch>=1.8.0;python_version<'3.10'",
+    "torch>=1.9.0;python_version<'3.10'",
     "torch>=2.0.0;python_version>='3.10' and python_version<'3.12'",
     "torch>=2.2.0;python_version>='3.12'",
     'tqdm>=4.29',
     'XlsxWriter>=1.2.8',
-    'rdt>=1.11.0',
-    'sdmetrics>=0.14.0',
-    'sdv>=1.12.0',
+    'rdt  @ git+https://github.com/sdv-dev/RDT@main',
+    'sdmetrics @ git+https://github.com/sdv-dev/SDMetrics@main',
+    'sdv @ git+https://github.com/sdv-dev/SDV@main',
```

Comment:
why is this needed?

Reply:
Otherwise, the builds were crashing:
https://github.com/sdv-dev/SDGym/actions/runs/8893490555/job/24420381562

I can try without SDV pointing to main, according to the error message it seems there is conflict between rdt, sdmetrics and sklearn.